my-dsl.md

```diff
@@ -2,37 +2,47 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[Zillions of Games](http://www.zillions-of-games.com/) is a board game scripting language.
 
 # Domain
 _Describe the language's domain in five words._
 
+Grid-Based Board Game Rules
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+When a *Zillions of Games*  file is loaded, the game engine creates a board game on the specified grid with the specified rules, and plays the game automatically. 
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+I would say that this language is very much a pure DSL. It literally only has one use-case, the creation of a board game that's played on a grid. If that's not purely domain-specific, I don't know what is. 
+I could see a similar kind of language, or an extension of this language being used to just draw things, but that's not the use case, and there are better tools for that anyway. 
```

Comment:
is the program extensible or usable with other programs? can you create rules, boards etc and then maybe add nicer UI or graphics elsewhere?